web/tests/copy_and_paste.test.cjs

```diff
@@ -239,4 +249,22 @@ run_test("paste_handler_converter", () => {
 
     // Pasting from Excel using ^⇧V should paste formatted text.
     assert.equal(copy_and_paste.paste_handler_converter(input), "     \n\n$ 20.00\n\n$ 7.00");
+
+    // Math block tests
+    for (const math_block_test of katex_tests.math_block_tests) {
+        input = math_block_test.input;
+        assert.equal(
+            copy_and_paste.paste_handler_converter(input),
+            math_block_test.expected_output,
+        );
+    }
+
+    // Inline Math Expression tests
+    for (const inline_math_expression_test of katex_tests.inline_math_expression_tests) {
+        input = inline_math_expression_test.input;
+        assert.equal(
+            copy_and_paste.paste_handler_converter(input),
+            inline_math_expression_test.expected_output,
+        );
```

Comment:
This seems to still not verify that rendering `original_markup` generates `input`. Can you try adding that in a follow-up PR?

Reply:
I'm not sure why that doesn't work, maybe you can ask in #frontend for help, but I don't want to block integration on that.